### DISCUSSION TITLE
Linux客户端无法向Windows或者Android发送文件

### DISCUSSION THREAD

**Discussion by See-Night** [localsend/localsend#746] [L-简体中文]
操作系统：Ubuntu 23.04
软件版本：1.11.1(35)

问题描述：
在同一局域网下，Linux 的 Localsend 客户端无法向 Windows 和 Android 发送文件，而 Windows 和 Android 之间的发送是正常的。
三端客户端版本一致，Linux 的防火墙也开放了 53317 端口。
![image](https://github.com/localsend/localsend/assets/45226780/5f19ee81-29a9-4d86-8054-a294c4b47656)
telnet 手机的端口也没有问题
![image](https://github.com/localsend/localsend/assets/45226780/9da8a088-ead9-456e-ad32-07dda3cf6afe)
![image](https://github.com/localsend/localsend/assets/45226780/f18fb31f-b32f-4f0d-8383-f5a84e30e1cd)

错误信息：

```
DioException [unknown]: null
Error: HandshakeException: Connection terminated during handshake
```

![image](https://github.com/localsend/localsend/assets/45226780/57118ae1-b485-4b31-a817-d0f88cf7ac9c)

**Comment by 2508388310** [comment]
有解决方案吗

**Reply by See-Night** [reply]
emmmm，我当时发现是我自己的代理问题，我把网段加进非代理名单就搞定了
